### ISSUE TITLE
Comms Consoles now print out empty Centcomm reports

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#2783]
Just experienced now on build 3989

The reports look like this when read: 

![image](https://user-images.githubusercontent.com/20813925/73346019-d9078180-42d0-11ea-9119-4130c9ce75e9.png)

**Comment by daalta** [comment]
Might have something to to with the chat changes. I'll check it out.

Relevant files: 
https://github.com/unitystation/unitystation/blob/develop/UnityProject/Assets/Scripts/Managers/CentComm.cs
https://github.com/unitystation/unitystation/blob/develop/UnityProject/Assets/Scripts/Chat/Chat.cs

**Comment by daalta** [comment]
I'm not sure if this related, but all paper seems broken somehow. In your screenshot it's visible that the entire page is squeezed vertically.

From what I can tell this issue is unrelated to my recent chat changes. Check CentComm.cs to see how the report pages are generated: 
```

	private void SendReportToStation()
	{
		var commConsoles = FindObjectsOfType<CommsConsole>();
		foreach (CommsConsole console in commConsoles)
		{
			var p = Spawn.ServerPrefab(paperPrefab, console.transform.position, console.transform.parent).GameObject;
			var paper = p.GetComponent<Paper>();
			paper.SetServerString(CreateStartGameReport());
		}

		Chat.AddSystemMsgToChat(CommandUpdateAnnouncementString(), MatrixManager.MainStationMatrix);

		SoundManager.PlayNetworked("Notice1", 1f);
		SoundManager.PlayNetworked("InterceptMessage", 1f);
	}
```

**Comment by JesterX666** [comment]
It has something to do with being "Renamable" type.  Interracting with a component of that type show this dialog instead of the real interraction.

**Comment by JesterX666** [comment]
Fixed by merge #2798
